### ISSUE TITLE
Required Radio Button field not respecting default value

### ISSUE THREAD

**Issue by braican** [verbb/formie#1221]
### Describe the bug

When creating a form with a required Radio Button field, any default default values are not being respected. It looks like in the latest plugin update, the `mergeAttributes` method in the `Html` helper class moved to using the `array_unique` function when merging attributes:

https://github.com/verbb/formie/blob/a6a58673b072d96691e758db1f2d19a0025a48bd/src/helpers/Html.php#L55

This causes a conflict in the case of a required Radio Button field that is checked, as the `array_unique` call would filter out the `checked` item from this array:

```php
$attributes = [
  'type' => 'radio',
  'id' => 'some-id-here',
  'name' => 'fields[fieldName]',
  'required' => 1,
  'checked' => 1, // This gets filtered out since its value is the same as the `required` key
  'value' => 'On'
];
```

I don't have the background as to why the `array_unique` call was introduced in https://github.com/verbb/formie/commit/4a83ff23dcab6937755d9c973df971ab85d2ebe9, but removing that function call there fixes the issue.

### Steps to reproduce

1. Create a new form.
2. Add a Radio Button field and set the field as required. Add some options and select one of them as the default.
3. Save the form and render it on a template. Notice that the option you selected as the default is _not_ checked.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.3.5

### Plugin version

2.0.19

### Multi-site?

No

**Comment by engram-design** [comment]
Should be fixed already for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.20](https://github.com/verbb/formie/releases/tag/2.0.20)
